### DISCUSSION TITLE
PGroonga 2.4.4 リリース!

### DISCUSSION THREAD

**Discussion by HashidaTKS** [pgroonga/pgroonga#287] [Releases]
## PGroonga（ぴーじーるんが） 2.4.4リリース

PostgreSQLで高速日本語全文検索をできるようにする[PGroonga](https://pgroonga.github.io/ja/)の2.4.4をリリースしました！

### 改良

* [[`pgroonga_wal_apply` 関数](https://pgroonga.github.io/ja/reference/functions/pgroonga-wal-apply.html)] エラーログにインデックスの情報を追加しました

### [[`pgroonga_wal_apply` 関数](https://pgroonga.github.io/ja/reference/functions/pgroonga-wal-apply.html)] エラーログにインデックスの情報を追加しました

この修正により、PGroongaのWALレコードの適用に失敗したインデックスを確認できるようになりました。

## まとめ

新規ユーザーの方は、[PGroongaについて](https://groonga.org/ja/blog/2017/10/10/pgroonga-2.0.2.html#pgroonga)も参照してください。

PostgreSQLで高速に日本語全文検索をしたいという方はPGroongaを使ってガンガン検索してください！

**Comment by askdkc** [comment]
macOS用にHomebrewにPR送っておきました

https://github.com/Homebrew/homebrew-core/pull/121843

**Reply by komainu8** [reply]
ありがとうございます！
